new-integration-tests/pom.xml

```diff
@@ -81,6 +81,11 @@
           <artifactId>commons-io</artifactId>
           <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>net.sourceforge.htmlunit</groupId>
+          <artifactId>htmlunit</artifactId>
+          <version>2.40.0</version>
+        </dependency>
```

Comment:
Add <scope>test</scope>... This keeps people happy who review our source because we won't have created a business approval to ship this jar.

Reply:
Added scope as test